docs/source/configuration.rst

```diff
@@ -335,6 +335,19 @@ This relationship can further be maintained in the JSON-LD structured data using
             ssn: "http://www.w3.org/ns/ssn/"
             Datastream: sosa:isMemberOf
 
+Sometimes, the JSON-LD desired is more complicated than can be achieved by simply aliasing properties using a
+context. In thise case, it is possible to specify a jinja2 template. When ``template`` is defined for a feature
+collection, the json-ld prepared by pygeoapi will be used to render a jinja2 template. pygeoapi provides a default
+template, which will render the unmodified json-ld document.
+
+.. code-block:: yaml
+
+    template: jsonld/base.jsonld
```

Comment:
Given the functionality is scoped to a single feature collection item, what about renaming this directive to `item_template`?

Thinking more, what about something like:

```yaml
item_templates:
    - jsonld: /path/to/file.jsonld
    - xml: /path/to/file.xml
```

Reply:
Note: whichever way we decide, this optional property will need to be added to the config schema in `pygeoapi/schemas/config/pygeoapi-config-0.x.yml`.

Reply:
At the moment, `template:  /path/to/file.xml` renders at `.../items/{id}?f=jsonld`. I like your suggestion ... but does that not encroach a bit upon the formatter provider? How does pygeoapi serve the correct `f=__` for defined templates?